changelog.txt

```diff
@@ -1,3 +1,7 @@
+Version 1.20.1
+=============
+- Fix Base64URL decoding bug (#938)
```

Comment:
Consider moving to an automated changelog, like MSAL Go https://github.com/AzureAD/microsoft-authentication-library-for-go

Reply:
(non blocking comment)